db/migrate/20241023112339_change_object_type_to_json_for_versions.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeObjectTypeToJsonForVersions < ActiveRecord::Migration[7.1]
+  def up
+    change_column :versions, :object, :json, using: "object::text::json"
```

Comment:
Are all existing version `object` values valid JSON?

Reply:
I've checked all the records in prod, they are valid JSON. (non-nil ones)

Reply:
can we check on all envs please?